src/main/java/by/exadel/internship/entity/user/TechUser.java

```diff
@@ -0,0 +1,9 @@
+package by.exadel.internship.entity.user;
+import lombok.NoArgsConstructor;
+
+
+@NoArgsConstructor
+public class TechUser  {
```

Comment:
As far as I remember we decided to remove this class.

Reply:
Sorry for my misunderstanding, I've dropped it. Done